editoast/src/views/train_schedule.rs

```diff
@@ -1105,7 +1107,7 @@ mod tests {
         )
     }
 
-    fn mocked_core_pathfinding_sim_and_proj(train_id: i64) -> MockingClient {
+    pub fn mocked_core_pathfinding_sim_and_proj(train_id: i64) -> MockingClient {
```

Comment:
```suggestion
    pub(in crate::views) fn mocked_core_pathfinding_sim_and_proj(train_id: i64) -> MockingClient {
```

We may want to use a pattern that's easy to grep for when we'll have reworked the module tree. This way we can get all placed where the visibility may need to change. Wdyt?